testnets/testnets.md

```diff
@@ -20,11 +20,12 @@ Below is the [demonet configuration](https://github.com/CosmWasm/testnets/blob/m
 You can find the other networks configurations at [CosmWasm/testnets](https://github.com/CosmWasm/testnets).
 
 ```sh
-export CHAIN_ID=testing
-export TESTNET_NAME=demo-10
-export RPC=https://rpc.demo-10.cosmwasm.com:443
-export FAUCET=https://faucet.demo-10.cosmwasm.com
-export SEED_NODE=1445f84f409745c554c03557e826edc9757b941a@p2p.demo-10.cosmwasm.com:26656
+export CHAIN_ID=cosmwasm-coral
+export TESTNET_NAME=coral
```

Comment:
Where is this used? The full marketing name is Coralnet

Reply:
It's used in setup scripts. The directory of coral config file in genesis: https://github.com/CosmWasm/testnets/tree/master/coral

Reply:
okay, but here it is coralnet: https://github.com/CosmWasm/testnets/blob/master/coral/defaults.env#L2

Reply:
Apparently we had a miscommunication there. Will not happen again.

Reply:
No worries :) I still wonder if we really need this variable at all but this is out of scope for this PR

Reply:
We can come back to and refactor it on the next testnets.